dev-packages/browser-integration-tests/suites/tracing/request/fetch-trace-header-merging/test.ts

```diff
@@ -32,14 +32,16 @@ async function assertRequests({
       });
   });
 
+  expect(requests).toHaveLength(2);
+
   requests.forEach(request => {
     const headers = request.headers();
 
     // No merged sentry trace headers
     expect(headers['sentry-trace']).not.toContain(',');
 
     // No multiple baggage entries
-    expect(headers['baggage'].match(/sentry-trace_id/g) ?? []).toHaveLength(1);
+    expect(headers['baggage'].match(/sentry-release/g) ?? []).toHaveLength(1);
```

Comment:
this seems harmless but it demonstrates the behaviour change for reused request options:

previously, we'd simple disregard what sentry values were in the request options and always overwrite them. Now:
- we still don't blindly append our values to pre-existing ones
- but instead, we do nothing.


Result: All requests in this test sharing the same request options have the same trace parameters.

I think this is fine but it's still worth noting.